clang/test/3C/cast.c

```diff
@@ -27,28 +27,52 @@ void bar(void) {
 #include<stdlib.h>
```

Comment:
These seem to be test updates that weren't included in some earlier PR. I don't object to including them here, just making a note of it.

Reply:
That's probably from running the scripts. They changed more than I expected and I didn't know if people usually reset those changes before submitting. Since this PR was partly to display the changes I didn't know enough about, I left them all in.